src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -36,36 +41,37 @@
     private boolean isInstructorAllowedToDelete;
     private boolean isInstructorAllowedToEdit;
 
-    public FeedbackResponseCommentRow(FeedbackResponseCommentAttributes frc, String giverDisplay) {
+    public FeedbackResponseCommentRow(FeedbackResponseCommentAttributes frc, String giverDisplay,
+            Map<String, String> instructorEmailNameTable) {
+        this.instructorEmailNameTable = instructorEmailNameTable;
         this.commentId = frc.getId();
         this.giverDisplay = giverDisplay;
         this.createdAt = TimeHelper.formatDateTimeForComments(frc.createdAt);
-        this.editedAt = frc.getEditedAtText("Anonymous".equals(giverDisplay));
         this.commentText = frc.commentText.getValue();
+        setCommentGiverNameFromEmail(giverDisplay);
+        setCommentLastEditorNameFromEmail(frc.lastEditorEmail);
```

Comment:
In addition, it looks like `commentLastEditorName` needs not be a field. If its usage is only at `setEditedAtText`, the operations can be moved there instead, i.e. the method `setEditedAtText` will first process the last editor name, then the rest.